src/OpenSage.Game/Logic/Object/GameObject.cs

```diff
@@ -1476,6 +1476,8 @@ public void Persist(StatePersister reader)
             reader.PersistUInt32(ref teamId);
             Team = GameContext.Game.TeamFactory.FindTeamById(teamId);
 
+            Owner = Team.Template.Owner;
```

Comment:
This is unrelated to the PR. I just found that game objects are assigned to the wrong owner when loading a `.sav` file.